packages/common/src/visualizations/SqlRunnerResultsTransformer.ts

```diff
@@ -349,18 +355,28 @@ export class SqlRunnerResultsTransformer
     }
 
     defaultPieChartFieldConfig(): SqlTransformPieChartConfig | undefined {
-        const firstCategoricalColumn = this.columns.find(
+        const activeColumns = this.columns.filter((column) => !column.isNull);
+
+        const firstCategoricalColumn = activeColumns.find(
             (column) => column.type === DimensionType.STRING,
         );
 
-        const firstNumericColumn = this.columns.find(
+        const firstDateColumn = activeColumns.find((column) =>
+            [DimensionType.DATE, DimensionType.TIMESTAMP].includes(column.type),
+        );
+
+        const firstNumericColumn = activeColumns.find(
             (column) => column.type === DimensionType.NUMBER,
         );
 
-        const groupFieldIds = firstCategoricalColumn?.reference
-            ? [firstCategoricalColumn.reference]
-            : undefined;
-        const metricId = firstNumericColumn?.reference;
+        let groupFieldIds: string[] | undefined;
+        if (firstCategoricalColumn?.reference) {
+            groupFieldIds = [firstCategoricalColumn.reference];
+        } else if (firstDateColumn?.reference) {
+            groupFieldIds = [firstDateColumn.reference];
```

Comment:
yes. example: grouping by year/month

Reply:
Even though possible to have that option, I wouldn't add a date column as the default group.

Reply:
Thanks, this makes sense! 

Since we can't **yet** add a metric: https://github.com/lightdash/lightdash/issues/10940 to a pie chart, I will then remove this default option, keep the select empty, but still allow it in case the user wants to select it